docs/discovery-search.html

```diff
@@ -112,13 +88,41 @@ <h3>
                     </g>
                   </svg>
                 </button>
-              </div>
-
-            </div>
+              </div> -->
           </div>
         </div>
       </div>
-    </div>
+    </div><!-- /nypl-page-header -->
+    <div class="nypl-sorter-row">
+      <div class="nypl-full-width-wrapper">
+        <div class="nypl-row">
+          <div class="nypl-column-full">
+          <div class="nypl-results-summary" aria-live="assertive" aria-atomic="true">
+            <h2>Displaying 1-50 of 843 results for keyword "New York Waterways"</h2>
+          </div>
+          <div class="nypl-results-sorting-controls">
+            <div class="nypl-results-sorter">
+              <div class="nypl-select-field-results">
+                <label for="sort-by-label">Sort by</label>
+                <form action="#" method="POST">
+                  <span class="nypl-omni-fields">
+                    <strong>
+                      <select id="sort-by-label" name="sort_scope">
+                      <option selected="selected" value="relevance">relevance</option>
+                      <option value="title_asc">title (a - z)</option><option value="title_desc">title (z - a)</option>
+                      <option value="date_asc">date (old to new)</option>
+                      <option value="date_desc">date (new to old)</option>
+                    </select>
+                  </strong>
+                </form>
+              <button id="sort-submission" type="submit" class="">Submit Query</button>
+            </div> <!-- nypl-select-field-results -->
+          </div><!-- /nypl-results-sorter -->
+        </div><!-- /nypl-results-sorting-controls -->
+      </div>
```

Comment:
`<!-- /nypl-column-full -->`

Reply:
This can also be indented, which means some of the DOM in the `<select>` are not indented properly.